README.md

```diff
@@ -1,2 +1,5 @@
 # week2day1
 Project Review
+
+
```

Comment:
- [ ] This is an inline comment

Reply:
- [ ] [REQUIRED] Hi @Yordinia,  from the project requirement you should use at least one inline comment. Please check the following link to see how to do [inline comment](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)